### DISCUSSION TITLE
Ethernet RJ45 TCP/IP Relay Remote Control

### DISCUSSION THREAD

**Discussion by arieltools** [tagyoureit/nodejs-poolController#676] [Nixie] [answered]
Has anyone tried something like this:

https://www.amazon.com/dp/B076CQ4CYL?ref_=cm_sw_r_ud_dp_MAA73EHZ4DZQBXP64372

**Comment by rstrouse** [comment, marked as answer]
njsPC should be able to control that using a custom interface file that sends the http requests.  However, REM will maintain latches for comms failures and dropouts.  I have used the Sequent hats and stacked 16 relays on a single pi.   You can put up to 64 on the i2c bus.
